CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.18.0
```

Comment:
The convention is to use `1.18.0-dev` before release

Reply:
ok.